autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -221,7 +225,7 @@ def load(cls, path, file_prefix="", reset_paths=False, verbose=True):
             return obj
 
     # TODO: Consider disabling feature pruning when num_features is high (>1000 for example), or using a faster feature importance calculation method
-    def compute_feature_importance(self, X, y, features_to_use=None, preprocess=True, **kwargs):
+    def compute_feature_importance(self, X, y, features_to_use=None, preprocess=True, silent=False, **kwargs):
         sample_size = 10000
         if len(X) > sample_size:
```

Comment:
Should this always do subsampling, or only when user did not provide a dataset?

Reply:
Added